examples/Cargo.toml

```diff
@@ -11,6 +11,9 @@ futures-channel = "0.3"
 futures-util = "0.3"
 once_cell = "1.2.0"
 
+[dependencies.glib]
+path = "../glib"
```

Comment:
Good that this became an assertion instead of silently doing random things :)

Reply:
Except glib is re-exported by gtk, not sure if that's a good thing actually :)

Reply:
The problem with that is that it worked accidentally before. If the dependency was not found then it was using simply the name `glib`. This required `glib` manually to be put into the scope by the user, and it wouldn't even be guaranteed that it's the correct `glib`.